manager-parent/manager-core/src/main/resources/db/migration/0.2/V0.2.1__create_v2_db.sql

```diff
@@ -0,0 +1,75 @@
+CREATE TABLE BRIDGE_V2 (
+    id                 varchar(255) NOT NULL PRIMARY KEY,
+
+    customer_id        varchar(255) NOT NULL,
+    organisation_id    varchar(255) NOT NULL,
+    owner              varchar(255) NOT NULL,
+    subscription_id    varchar(255) NOT NULL,
+
+    name               varchar(255) NOT NULL,
+    cloud_provider     varchar(255) NOT NULL,
+    region             varchar(255) NOT NULL,
+
+    endpoint           varchar(255),
+    shard_id           varchar(255) NOT NULL,
+
+    submitted_at       timestamp NOT NULL,
+    published_at       timestamp,
+
+    error_id           integer,
+    error_uuid         varchar(255),
```

Comment:
Personally, now we have `Condition`'s I'd be tempted to kill this off.

It was used to propagate "(failure) reasons" to the User and is largely constructed from `Condition` messages.

@robpblake @lucamolteni WDYT?

Reply:
Gone :rocket: